src/ray/object_manager/plasma/stats_collector.h

```diff
@@ -60,6 +60,8 @@ class ObjectStatsCollector {
  private:
   friend struct ObjectStatsCollectorTest;
 
+  int64_t num_bytes_in_memory_ = 0;
```

Comment:
Do we need a new counter? It seems you can derive this from some of the counters below.